Source/FluidOscServer.cpp

```diff
@@ -51,6 +50,7 @@ void FluidOscServer::oscMessageReceived (const OSCMessage& message) {
     if (msgAddressPattern.matches({"/plugin/load"})) return loadPluginPreset(message);
     if (msgAddressPattern.toString().startsWith("/plugin/sampler")) return handleSamplerMessage(message);
     if (msgAddressPattern.matches({"/audiotrack/select"})) return selectAudioTrack(message);
+    if (msgAddressPattern.matches({"/audiotrack/insert"})) return insertWaveSample(message);
```

Comment:
I think this should be `/audiotrack/insert/file` or maybe `/audiotrack/insert/sample` or perhaps `/audiotrack/insert/wav`